Source/WebCore/rendering/RenderFlexibleBox.h

```diff
@@ -198,7 +198,7 @@ class RenderFlexibleBox : public RenderBlock {
     LayoutUnit computeFlexBaseSizeForChild(RenderBox& child, LayoutUnit mainAxisBorderAndPadding, bool relayoutChildren);
     void maybeCacheChildMainIntrinsicSize(RenderBox& child, bool relayoutChildren);
     void adjustAlignmentForChild(RenderBox& child, LayoutUnit);
-    ItemPosition alignmentForChild(const RenderBox& child) const;
+    ItemPosition alignmentForChild(const RenderBox& child, OverflowAlignment* safety = nullptr) const;
```

Comment:
We don't use arguments as out values generally. We should return a `std::pair` or an ad-hoc type here if necessary.

Reply:
The reason I didn't do that is that [most callers](https://searchfox.org/wubkat/search?q=RenderFlexibleBox%3A%3AalignmentForChild&path=&case=true&regexp=false) don't need the safety valu, and many of them are using this function inline in an expression to check the ItemPosition value so it would be more awkward for them to use a complex return type. I can switch it over to return a synthesized StyleSelfAlignmentData or something like that, but, that's why I went with this approach.

Reply:
OK, went with a separate function for the oveflow alignment value. The lookup logic isn't that complicated so it should be OK to duplicate it.